### ISSUE TITLE
Setting to clear translation after syncing when the source text of a unit changes

### ISSUE THREAD

**Issue by catadumitru** [rvanbekkum/vsc-xliff-sync#64]
Is there a setting that whenever the source text changes the target is removed?
I know this was the behavior in the past.
I would rather have no translation than one that needs adaptation.

**Comment by rvanbekkum** [comment]
Hi @catadumitru,
Thank you for filing this issue.

The behaviour has always been like this since version 0.2.5:
```
[0.2.5] 22-08-2019
If a change in the source-text of a trans-unit is detected during synchronization, then the target-node/translation of the trans-unit in the target file will be tagged with needs-adaptation. Also an "XLIFF Sync" note will be added to the trans-unit to clarify that a change of the source text was detected.
```
But, if no matching trans-unit can be found in the target file (depending on your `xliffSync.findBy...`-settings), then the trans-unit will be treated as a new trans-unit and the target is marked with `state="needs-translation"`, also see [Usage](https://github.com/rvanbekkum/vsc-xliff-sync#usage).

I could make a setting for what you suggested and include that in a next version 😉, i.e., if source text changes, then clear the translation.

**Comment by catadumitru** [comment]
I think what I needed was to fix my settings and use xliffSync.findByXliffGeneratorNoteAndSource
It can be that settings in VSCode get reset when reinstalling the app.

Thank you for your help and keep up the good work!

**Comment by rvanbekkum** [comment]
Changes for this are shipped in v0.7.0!
